packages/ui/package.json

```diff
@@ -33,37 +33,37 @@
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.0",
     "djb2a": "^2.0.0",
-    "framer-motion": "^11.0.8",
+    "framer-motion": "^11.0.22",
     "humanize-duration": "^3.31.0",
-    "lucide-react": "^0.354.0",
+    "lucide-react": "^0.363.0",
     "react-dom": "^18.2.0",
     "react-json-view-lite": "^1.3.0",
     "react-loader-spinner": "^6.1.6",
     "react-router-dom": "^6.22.3",
     "sonner": "1.4.3",
```

Comment:
Sonner has an update, but the latest version has a bug: https://github.com/emilkowalski/sonner/issues/378. We need to wait a bit one this one.